app/uk/gov/hmrc/fileupload/testonly/TestOnlyController.scala

```diff
@@ -51,7 +52,7 @@ class TestOnlyController(fileRepo: FileRepository, envelopeRepo: EnvelopeReposit
   }
 
   private def cleanupEnvelopeAndFiles: Future[(WriteResult, List[WriteResult])] = {
-    envelopeRepo.removeAll() zip fileRepo.removeAll()
+    envelopeRepo.removeAll() zip fileRepo.clear(Duration.ZERO)
```

Comment:
I think we should inject the functions as we do for all the other controllers instead of the repository. As a consequence we shouldn't create new instances of the repositories and reuse the exitsiting ones for the test.